<launch>
	<node pkg="nodelet" type="nodelet" name="standalone_nodelet" args="manager" output="screen"/>

	<node pkg="nodelet" type="nodelet" name="RaspiCamNd_orange" args="load raspicam_nd/RaspiCamNd standalone_nodelet">
		<remap from="image_raw" to="$(optenv IMAGE /ow_camera/image_raw)" />
		<param name="fps" type="int" value="5"/>
--		<param name="color_mode" type="str" value="rgb8"/>
		<param name="camera_info_url" type="str" value="file://$(find raspicam_nd)/pi_camera.yaml" />

		<param name="width" type="int" value="320"/>
		<param name="height" type="int" value="240"/>

<!--
		<param name="width" type="int" value="640"/>
		<param name="height" type="int" value="480"/>
-->

<!--
		<param name="width" type="int" value="2560"/>
		<param name="height" type="int" value="1920"/>
-->

	</node>

</launch>


